docs/cluster-management/configure-buildcluster-queue-worker.md

```diff
@@ -697,19 +758,53 @@ Configure Rabbitmq definitions using Rabbitmq admin UI **manually** or use **Imp
       "destination_type": "queue",
       "routing_key": "ClusterAdlr",
       "arguments": {}
+    },
+    {
+      "source": "build",
+      "vhost": "screwdriver",
+      "destination": "ClusterARetry",
+      "destination_type": "queue",
+      "routing_key": "ClusterARetry",
+      "arguments": {}
+    },
+    {
+      "source": "build",
+      "vhost": "screwdriver",
+      "destination": "ClusterBRetry",
+      "destination_type": "queue",
+      "routing_key": "ClusterBRetry",
+      "arguments": {}
+    },
+    {
+      "source": "build",
+      "vhost": "screwdriver",
+      "destination": "ClusterARetrydlr",
+      "destination_type": "queue",
+      "routing_key": "ClusterARetrydlr",
+      "arguments": {}
+    },
+     {
+      "source": "build",
+      "vhost": "screwdriver",
+      "destination": "ClusterBRetrydlr",
+      "destination_type": "queue",
+      "routing_key": "ClusterBRetrydlr",
+      "arguments": {}
     }
   ]
 }
 ```
 
-Note:
-1. Queues suffixed with dlr are deadletter queues. We use rabbitmq in-built deadletter queue mechanism for a retry with delay in case of errors.
-   Deadletter queues are used in case of any [error](https://github.com/screwdriver-cd/buildcluster-queue-worker/blob/master/index.js#L118)
-   in consuming the message and pushing to Kubernetes cluster for build processing. When a message is `nack'd` it goes to dlr queues via deadletter
-   routing key configuration and re-pushed to actual queue after a delay of 5s (per below configuration).
+Note: 
+1. Queues suffixed with dlr are deadletter queues. We use rabbitmq in-built deadletter queue mechanism for a retry with delay in case of errors. 
```

Comment:
```suggestion
Note:
1. Queues suffixed with `dlr` are deadletter queues. We use the built-in rabbitmq deadletter queue mechanism for a retry with delay in case of errors.
```